python/chemiscope/sphinx/directive.py

```diff
@@ -40,7 +39,8 @@ def run(self):
             source_path = os.path.dirname(source) + "/"
 
             # Path to the saved dataset in the .rst files folder
-            rst_file_path = source_path + self.options.get("filepath")
+            dataset_rel_path = self.content[0].strip()
```

Comment:
what happens if the path is absolute? I think sphinx interpret absolute path as relative to the root of the documentation

Reply:
I'm fine with this, but then we need a check in the code that raise an exception if the user is giving an absolute path here.

Reply:
Ok, let me add it quickly !

Reply:
well, you'll have to add it in the follow-up PR adding sphinx-gallery support, since this one is already merged =)